code/game/objects/items.dm

```diff
@@ -1024,3 +1021,13 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 /// Changes the speech verb when wearing this item if a value is returned
 /obj/item/proc/change_speech_verb()
 	return
+
+/// Plays a sound when used.
+/obj/item/proc/play_sound(atom/target, volume = use_volume)
```

Comment:
This is far too generic and may often be confused with `/playsound` use something like `play_use_sound` instead